code/game/objects/items/tools/wirecutters.dm

```diff
@@ -6,6 +6,9 @@
 	inhand_icon_state = "cutters"
 	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+
+	greyscale_config = /datum/greyscale_config/wirecutters
```

Comment:
This seems like a ticking bomb bug to me, since it's unclear the non-random subtypes won't get colorized only because of unset default color on parent type.